open-hackathon/src/hackathon/enum.py

```diff
@@ -35,6 +35,11 @@ class EmailStatus:
     NonPrimary = 0
 
 
+class ReservedUser:
+    DefaultUserID = -1
+    DockerVM = -2
```

Comment:
plz add more comments here in case we forget its design purpose some months later